src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -86,7 +86,7 @@
     <RuntimeGroup Include="fedora">
       <Parent>linux</Parent>
       <Architectures>x64;arm64</Architectures>
-      <Versions>23;24;25;26;27;28;29;30;31;32;33;34;35;36;37;38</Versions>
+      <Versions>36;37;38</Versions>
```

Comment:
Fedora 35 and previous are EOL 2022-12-13 (https://docs.fedoraproject.org/en-US/releases/eol/).